utest/api/test_languages.py

```diff
@@ -53,7 +53,7 @@ def test_standard_languages_have_code_and_name(self):
             assert cls.name
 
     def test_standard_language_doc_formatting(self):
-        added_in_rf60 = {'bg', 'bs', 'cs', 'de', 'en', 'es', 'fi', 'fr', 'hi',
+        added_in_rf60 = {'ar', 'bg', 'bs', 'cs', 'de', 'en', 'es', 'fi', 'fr', 'hi',
```

Comment:
Please remove `ar` here. This set contains the languages that were added in RF 6.0 when the localization support was implemented. The test verifies that their docstring contains only the language name and that all the others contain the `New in Robot Framework ...` note.